quarkchain/cluster/shard_state.py

```diff
@@ -1848,6 +1843,7 @@ def _posw_info(self, block: MinorBlock) -> Optional[PoSWInfo]:
                 self.env.quark_chain_config.genesis_token,
             ),
             block_cnt,
+            stake_per_block,
```

Comment:
I would suggest using a named argument (stake_per_block=stake_per_block) since the definition of the function assumes it is named.